Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -0,0 +1,254 @@
+        loginStackView.spacing = 10
+        stackView.insertArrangedSubview(loginStackView, at: 2)
+        stackView.setCustomSpacing(20, after: subtitleLabel)
+        stackView.setCustomSpacing(40, after: loginStackView)
+        loginStackView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+        }
+
+        setUpPrivacyStatementButton()
+        setUpErrorView()
+        setUpNetidViews()
+        setUpPasswordViews()
+        setUpSkipButton()
+    }
+
+    private func setUpPrivacyStatementButton() {
+        privacyStatementButton = UIButton(type: .system)
+        privacyStatementButton.setTitle("Privacy Statement", for: .normal)
+        privacyStatementButton.setTitleColor(.white, for: .normal)
+        privacyStatementButton.titleLabel?.font = .preferredFont(forTextStyle: .subheadline)
+        privacyStatementButton.setTitleColor(.black, for: .highlighted)
+        privacyStatementButton.addTarget(self,
+                                         action: #selector(privacyStatementButtonPressed(_:)),
+                                         for: .touchUpInside)
+        loginStackView.addArrangedSubview(privacyStatementButton)
+    }
+
+    @objc private func privacyStatementButtonPressed(_ sender: UIButton) {
+        let privacyStatementViewController = BRBPrivacyStatementViewController()
+        self.present(privacyStatementViewController, animated: true, completion: nil)
+    }
+
+    private func setUpErrorView() {
+        errorView = BRBLoginErrorView(frame: .zero)
+        errorView.isCollapsed = true
+        loginStackView.addArrangedSubview(errorView)
+    }
+
+    private func setUpNetidViews() {
+        netidPrompt = UILabel(frame: .zero)
+        netidPrompt.text = "NetID"
+        netidPrompt.textColor = .white
+        netidPrompt.font = .preferredFont(forTextStyle: .headline)
+        loginStackView.addArrangedSubview(netidPrompt)
+
+        netidTextField = UITextField(frame: .zero)
+        netidTextField.textColor = UIColor(white: 225.0 / 255.0, alpha: 1.0)
+        netidTextField.attributedPlaceholder = NSAttributedString(string: "Type your NetID (e.g. abc123)",
+        attributes: [NSAttributedString.Key.foregroundColor: UIColor(white: 225.0 / 255.0, alpha: 1.0)])
+        netidTextField.font = .preferredFont(forTextStyle: .body)
+        netidTextField.autocapitalizationType = .none
+        netidTextField.tintColor = UIColor(white: 225.0 / 255.0, alpha: 1.0)
+        netidTextField.autocorrectionType = .no
+        loginStackView.addArrangedSubview(netidTextField)
+
+        let netidSeparator = UIView()
+        netidSeparator.backgroundColor = UIColor(white: 225.0 / 255.0, alpha: 1.0)
+        loginStackView.addArrangedSubview(netidSeparator)
+        netidSeparator.snp.makeConstraints { make in
+            make.height.equalTo(1)
+        }
+    }
+
+    private func setUpPasswordViews() {
+        passwordPrompt = UILabel(frame: .zero)
```

Comment:
`UILabel()`, same for all the other views. you. don't need to say `frame: _` if you're passing in `.zero`

Reply:
I know, these were pulled directly from BRBLoginViewController